resources/views/partials/nav.blade.php

```diff
@@ -13,52 +13,61 @@
 					<ul class="nav navbar-nav">
-						
-						@can("viewAny", \App\Participant::class)						
+
+						@can("viewAny", \App\Participant::class)
 						<li class="{{ substr(Request::path(),0,12) == 'participants' ? 'active' : ''}}"><a href="{{ url('/participants') }}">Deelnemers</a></li>
 						@endcan
-						@can("viewAny", \App\Member::class)						
+						@can("viewAny", \App\Member::class)
 						<li class="{{ substr(Request::path(),0,7) == 'members' ? 'active' : ''}}"><a href="{{ url('/members') }}">Leden</a></li>
 						@endcan
 						@can("viewAny", \App\Event::class)
 						<li class="{{ substr(Request::path(),0,6) == 'events' ? 'active' : ''}}"><a href="{{ url('/events') }}">Evenementen</a></li>
 						@endcan
-						@can("viewAny", \App\Location::class)
-						<li class="{{ substr(Request::path(),0,9) == 'locations' ? 'active' : ''}}"><a href="{{ url('/locations') }}">Locaties</a></li>
-						@endcan
 
-						@can("viewAny", \App\Course::class)
-						<li class="{{ substr(Request::path(),0,7) == 'courses' ? 'active' : ''}}"><a href="{{ url('/courses') }}">Vakken</a></li>
-						@endcan
-						@can("viewAny", \App\Action::class)
-						<li class="{{ substr(Request::path(),0,7) == 'actions' ? 'active' : ''}}"><a href="{{ url('/actions') }}">Punten</a></li>
-						@endcan
-						@role(["member"])
-						<li class="{{ substr(Request::path(),0,5) == 'lists' ? 'active' : ''}}"><a href="{{ url('/lists') }}">Lijsten</a></li>
-						@endrole
-						<li class="{{ substr(Request::path(),0,6) == 'graphs' ? 'active' : ''}}"><a href="{{ url('/graphs') }}">Grafieken</a></li>
-						@can("viewAny", \App\User::class)
-						<li class="{{ substr(Request::path(),0,5) == 'users' ? 'active' : ''}}"><a href="{{ url('/users') }}">Gebruikers</a></li>
-						@endcan
-						@role(["board"])
-						<li class="{{ substr(Request::path(),0,5) == 'event-packages' ? 'active' : ''}}">
-							<a href="{{ url('/event-packages') }}">Pakketten</a>
-						</li>
-						@endrole
-						
 						@can("viewOwn", \App\Declaration::class)
 						<li class="{{ substr(Request::path(),0,5) == 'declaration' ? 'active' : ''}}">
 							<a href="{{ url('/declarations') }}">Declaraties</a>
 						</li>
 						@endcan
+
+						<li class="dropdown" >
+							<a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Overig <span class="caret"></span></a>
+							<ul class="dropdown-menu">
+
+								@can("viewAny", \App\Location::class)
+									<li class="{{ substr(Request::path(),0,9) == 'locations' ? 'active' : ''}}"><a href="{{ url('/locations') }}">Locaties</a></li>
+								@endcan
+
+								@role(["board"])
+								<li class="{{ substr(Request::path(),0,5) == 'event-packages' ? 'active' : ''}}">
+									<a href="{{ url('/event-packages') }}">Pakketten</a>
+								</li>
+								@endrole
+								<li class="{{ substr(Request::path(),0,6) == 'graphs' ? 'active' : ''}}"><a href="{{ url('/graphs') }}">Grafieken</a></li>
```

Comment:
Valt me nu op dat de Grafieken pagina niet onder een rol of capability valt. Maar ze zijn niet echt bedoeld voor deelnemers. Dus deze ook even onder de member role stoppen?